docs/content.zh/docs/deployment/memory/network_mem_tuning.md

```diff
@@ -24,147 +24,141 @@ specific language governing permissions and limitations
+当前，buffer debloating 仅在使用的 buffer 大小上设置上限。实际的 buffer 大小和 buffer 个数保持不变。这意味着 debloating 机制不会减少作业的内存使用。您应该手动减少 buffer 的大小或者个数。
 
-Furthermore, if you want to reduce the amount of buffered in-flight data below what buffer debloating currently allows, you might want to manually configure the number of buffers.
+此外，如果您想减少缓存的 in-flight 数据量使其低于 buffer debloating 当前允许的量，您可能需要手动的设置 buffer 的个数。
 
-## Network buffer lifecycle
+## Network buffer 生命周期
  
-Flink has several local buffer pools - one for the output stream and one for each input gate. 
-Each of those pools is limited to at most 
+Flink 有多个本地 buffer 池 —— 输出数据对应一个，每个 input gate 对应一个。
 
 `#channels * taskmanager.network.memory.buffers-per-channel + taskmanager.network.memory.floating-buffers-per-gate`
 
-The size of the buffer can be configured by setting `taskmanager.memory.segment-size`.
+buffer 的大小可以通过 `taskmanager.memory.segment-size` 来设置。
 
-### Input network buffers
+### 输入 network buffers
 
-Buffers in the input channel are divided into exclusive and floating buffers.  Exclusive buffers can be used by only one particular channel.  A channel can request additional floating buffers from a buffer pool shared across all channels belonging to the given input gate. The remaining floating buffers are optional and are acquired only if there are enough resources available.
+输入通道中的 buffer 被分为独占 buffer 和流动 buffer。每个独占 buffer 只能被一个特定的通道使用。
+一个通道可以从 input gate 的共享 buffer 池中申请额外的流动 buffer。剩余的流动 buffer 是可选的并且只有资源足够的时候才能获取。
 
-In the initialization phase:
-- Flink will try to acquire the configured amount of exclusive buffers for each channel
-- all exclusive buffers must be fulfilled or the job will fail with an exception
-- a single floating buffer has to be allocated for Flink to be able to make progress
+在初始阶段：
+- Flink 会为每一个输入通道获取配置数量的独占 buffer。
+- 所有的独占 buffer 都必须被满足，否则作业会抛异常失败。
+- 流动的 buffer 需要被分配给 Flink 才能推动进度。
 
-### Output network buffers
+### 输出 network buffers
 
-Unlike the input buffer pool, the output buffer pool has only one type of buffer which it shares among all subpartitions.
+不像输入 buffer 池，输出 buffer 池只有一种类型的 buffer 被所有的 subpartitions 共享。
 
-In order to avoid excessive data skew, the number of buffers for each subpartition is limited by the `taskmanager.network.memory.max-buffers-per-channel` setting.
+为了避免过多的数据倾斜，每个 subpartition 的 buffer 数量可以通过 `taskmanager.network.memory.max-buffers-per-channel` 来限制。
 
-Like the input buffer pool, the configured amount of exclusive buffers and floating buffers is only treated as recommended values. If there are not enough buffers available, Flink can make progress with only a single exclusive buffer per output subpartition and zero floating buffers.
+跟输入 buffer 池一样，配置的独占 buffer 和流动 buffer 只被当作推荐值。如果没有足够的 buffer，每个输出 subpartition 可以只使用一个独占 buffer 而没有流动 buffer 来推动 Flink 的进度。
```

Comment:
It seems a mistake. Please help to verify the doc. Thank you. @akalash

Reply:
@Myracle, @xintongsong, thanks for emphasizing this. It is indeed a mistake. I think the correct way to fix this it is exactly what you proposed - instead of `Like the input buffer pool` it should be `Unlike the input buffer pool`. 
The main idea is that if we have at least one buffer for each subpartition it will be enough to make a progress(It doesn't matter how many buffers we configured) while all exclusive buffers for the input channel have to be allocated on init and if there are less available buffers than we configured, Flink will fail.

@Myracle, can you fix this in English version in a separate commit as well? Or I can do it later.